packages/@icon-magic/generate/src/plugins/svg-to-raster.ts

```diff
@@ -186,12 +185,12 @@ export const svgToRaster: GeneratePlugin = {
           png: {
             name: assetName,
             path: `./${assetName}.png`,
-            imageset: imageset
+            imageset: imageset,
+            colorScheme: flavor.colorScheme
           },
           webp: {
             name: assetName,
-            path: `./${assetName}.webp`,
-            imageset: imageset
+            path: `./${assetName}.webp`
```

Comment:
Good to propagate the colorScheme and imageset values to webp as well, even though it isn't used anywhere as of now